### DISCUSSION TITLE
Markdown with frontmatter causing 404 when deployed (React + Vite)

### DISCUSSION THREAD

**Discussion by sdelessio** [community/community#146779] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My site is built with vite and React and I am deploying with github pages. It is a private repo. I'm using markdown files to populate my page content, which works locally. When I deployed my site, any page with frontmatter broke with a 404 error. If I just remove the frontmatter from the affected markdown and keep any markdown content, the file will load.

I've tried adding a .nojekyll file, though I'm not sure if it should be in the main branch or the gh-pages branch.

**Comment by sdelessio** [comment]
I solved this issue. The fix was that I had the .nojekyll file in the wrong place. It should not have been in the main branch. I added it directly to the gh-pages branch through the github UI using the 'add file' button and that resolved my issue.

**Reply by Pricer187** [reply]
Wasup
